files/build_templates/per_namespace/teamd.service.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=TEAMD container
+{% if multi_instance == 'true' %}
+Requires=swss@%i.service
```

Comment:
Why does teamd require swss if multi-ASIC, but not if single-ASIC?

Reply:
Fixed it. Does not require this dependency specifically for multi-asic.